microservices.yml

```diff
@@ -81,6 +81,35 @@ commands:
           command: |
             make push-image-to-ecr
 
+  push-built-image-to-prod-ecr:
+    description: Publish docker image to production ECR
+    steps:
+      - run:
+          name: Switch to PRODUCTION aws account
+          command: |
+            echo "export ORIGINAL_CIRCLE_BRANCH=$CIRCLE_BRANCH" >> $BASH_ENV
+            echo "export CIRCLE_BRANCH=prod" >> $BASH_ENV
+            . $BASH_ENV
+            echo "Manually set CIRCLE_BRANCH to $CIRCLE_BRANCH inorder to force-switch okta cli to production aws account"
+            rm -rf ~/.okta
+      - install-okta-aws-cli
+      - run:
+          name: Push Image to production ECR
+          command: |
+            echo "export ECR_APP_IMAGE_ORIGINAL=$ECR_APP_IMAGE" >> $BASH_ENV
+            echo "export ECR_APP_IMAGE=$(aws sts get-caller-identity | jq -r '.Account').dkr.ecr.us-west-2.amazonaws.com/devices" >> $BASH_ENV
+            . $BASH_ENV
+            make build tag push-image-to-ecr
+      - run:
+          name: Reset CircleCI branch env var
+          command: |
+            echo "export CIRCLE_BRANCH=$ORIGINAL_CIRCLE_BRANCH" >> $BASH_ENV
+            echo "export ECR_APP_IMAGE=$ECR_APP_IMAGE_ORIGINAL" >> $BASH_ENV
+            . $BASH_ENV
+            echo "Manually revert CIRCLE_BRANCH to $CIRCLE_BRANCH and ECR_APP_IMAGE to $ECR_APP_IMAGE_ORIGINAL inorder to switch okta cli back to non-prod aws account"
+            rm -r ~/.okta
+      - install-okta-aws-cli
```

Comment:
This new job (`push-image-prod`) uses the following flow:

1. Switch to prod aws account by manually setting `CIRCLE_BRANCH` & `ECR_APP_IMAGE` to production values
2. Publish docker image to `prod` aws account 
3. Cleanup environment changes; Switch back to non-prod account